core/src/main/resources/schema/oracle/oracle-create-schema.sql

```diff
@@ -55,3 +55,34 @@ BEGIN
   EXECUTE IMMEDIATE 'ALTER SEQUENCE EVENT_JOURNAL__ORDERING_SEQ INCREMENT BY 1 MINVALUE 0';
 END;
 /
+
+CREATE SEQUENCE DURABLE_STATE__GLOBAL_OFFSET_SEQ START WITH 1 INCREMENT BY 1 NOMAXVALUE
+/
+
+CREATE TABLE DURABLE_STATE
+(
+    GLOBAL_OFFSET         NUMERIC      NOT NULL,
+    PERSISTENCE_ID        VARCHAR(255) NOT NULL,
+    REVISION              NUMERIC      NOT NULL,
+    STATE_PAYLOAD         BLOB         NOT NULL,
+    STATE_SERIAL_ID       NUMBER(10) NOT NULL,
```

Comment:
minor format issue - can you add a space before 'NOT'?

Reply:
fixed